pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -826,6 +832,17 @@ self: super: {
   # https://github.com/aisamanra/config-ini/issues/12
   config-ini = dontCheck super.config-ini;
 
+  config-value = (doJailbreak (overrideCabal super.config-value (drv: {
+    buildTools = [
+      (super.callHackageDirect
+        {
+          pkg = "happy";
+          ver = "1.19.12";
+          sha256 = "0n1ri85hf1h9q5pwvfnddc79ahr9bk7hz8kirzrlyb81qzc3lpc9";
+        }
+        { })
```

Comment:
What is the problem with `happy`?  Does `config-value` need an older or a newer version?

In general, things using IFD won't work in nixpkgs because of hydra, so you won't be able to use `callHackageDirect` for this.

The `extra-packages` section in `configuration-hackage2nix.yaml` lets you define new packages that our Haskell stuff will generate and provide if you need some other version of `happy`.

Reply:
yeah I thought maybe I wasn't allowed to use that in nixpkgs. :( I suspected the extra-packages is what I should use. too. (I asked on IRC but didn't get a reply *shrug*)

```
  build-tool-depends:
    alex:alex   ^>= 3.2.4,
    happy:happy ^>= 1.19,
```

Reply:
I'll look into fixing the bound on happy to accept 1.20

Reply:
I clicked over to your repo to file an issue after my last comment here and I had a notification :rofl: :smile: @cdepillabout like I said glguy is on point! :heart: